### DISCUSSION TITLE
Restoring deleted services

### DISCUSSION THREAD

**Discussion by yudapedia** [seerge/g-helper#145] [Q&A] [answered]
I ran the debloat bat file https://raw.githubusercontent.com/seerge/g-helper/main/debloat.bat

Is there anyway to restore the deleted service? I tried installing Asus Control System Interface v3 but it cannot be installed as I have the latest version of the driver.

Thank you.

**Comment by seerge** [comment, marked as answer]
You need to uninstall Asus Control System Interface v3 it first :)

But since there is there is no "uninstall" option for that thing I just did a following :

1. Opened device manager -> System Devices -> Asus Control System Interface . Right click on it "uninstall" (check remove driver)
![Screenshot 2023-03-26 141503](https://user-images.githubusercontent.com/5920850/227775019-0584f9f0-7e38-4648-82a9-c61d2d6aeba1.png)

2. Reboot
3. Install fresh Asus Control System Interface v3 (along with all it's services)

But in general, it's a good point. So I have updated debloat script. Now it will disable services, instead of deleting them.
